agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -298,12 +299,14 @@ function install_k8s_agent {
       kubectl apply -f $IA_CONFIG_FILE --namespace=$NAMESPACE
     fi
 
-    AGENT_STRING="agent"
-    if [ ! -z "$AGENT_SLIM" ]; then
-        DAEMONSET_FILE="/tmp/sysdig-agent-slim-daemonset-v2.yaml"
-        AGENT_STRING="agent-slim"
-    else
+    if [ $AGENT_FULL -eq 1 ]; then
```

Comment:
for consistency, please change this to match the other input which is validated with `-z`

Reply:
nm, I see that AGENT_FULL should always equal 0 or 1